code/modules/status_system/statusEffects.dm

```diff
@@ -2926,14 +2926,17 @@
 				"Luck is on your side... wait, what...?"))
 			boutput(src, SPAN_NOTICE(msg))
 
-/datum/statusEffect/talisman_fault
-	id = "art_talisman_fault"
+/datum/statusEffect/talisman_held
+	id = "art_talisman_held"
 	unique = FALSE
 	visible = FALSE
 	effect_quality = STATUS_QUALITY_NEGATIVE
```

Comment:
this feels like an odd choice - just a copypaste thing or is there a reason for it?

Reply:
srry, could you clarify which line you're talking about?

Reply:
oh, it being `STATUS_QUALITY_NEGATIVE`

Reply:
Oh ok. Yeah I can change it to neutral if you think it would be better, I was considering artifact faults negative, bit awkward with the visual effect too in one status effect... dunno

Reply:
I'll just split it up into two

Reply:
> I'll just split it up into two

I instead have just made godmode block artifact faults from occurring entirely for the user

Reply:
okay cool